custom_components/tuya_local/devices/diivoo_dwv010.yaml

```diff
@@ -25,7 +25,7 @@ secondary_entities:
         type: string
         name: operation
   - entity: sensor
-    class: volume
+    class: total_increasing
```

Comment:
Sorry, it is confusing but there are two separate `class` attributes.
Device class (volume) goes here. State class (total_increasing) should go on the dp definition below.